<launch>

<arg name="aruco_markerSize" default="0.5"/> <!-- in meter -->
<arg name="eye" default="left"/>
<arg name="marker_frame" default="marker_frame"/>
<arg name="aruco_ref_frame" default="base_link"/> <!-- leave empty and the pose will be published wrt param parent_name -->
<arg name="aruco_camera_frame" default="camera_color_optical_frame"/>

<node pkg="aruco_ros" type="marker_publisher" name="aruco_marker_publisher">
        <remap from="/camera_info" to="/camera/color/camera_info" />
        <remap from="/image" to="/camera/color/image_raw" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size" value="$(arg aruco_markerSize)"/>
        <param name="reference_frame" value="$(arg aruco_ref_frame)"/>  <!--frame in which the marker pose will be refered -->
        <param name="camera_frame" value="$(arg aruco_camera_frame)"/>
</node>

<node pkg="leonardo_2" type="aruco_manager" name="aruco_manager" output="screen">

</node>

</launch>
